packages/core/src/App/Containers/__tests__/trade-notifications.spec.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { SwipeableNotification } from '@deriv/components';
+import { StoreProvider, mockStore, useStore } from '@deriv/stores';
+import { getCardLabels } from '@deriv/shared';
+import TradeNotifications from '../trade-notifications';
+
+type TMockedSwipeableNotificationProps = React.ComponentProps<typeof SwipeableNotification> & {
+    onUnmount?: () => void;
+    timestamp?: number | null;
+};
+
+jest.mock('@deriv/components', () => ({
+    ...jest.requireActual('@deriv/components'),
+    MobileWrapper: jest.fn(({ children }) => children),
+    SwipeableNotification: ({ children, onUnmount, timestamp }: TMockedSwipeableNotificationProps) => {
+        if (onUnmount) onUnmount();
+        return (
+            <div>
+                {children}
+                {!!timestamp && <div>Timestamp</div>}
+            </div>
+        );
+    },
+}));
+
+describe('TradeNotifications', () => {
+    const STATUS = {
+        OPEN: 'open',
+        COMPLETED: 'completed',
+    };
+    const getTradeNotification = (status: string) => ({
+        id: `${Math.random()}_${status}`,
+        status,
+        timestamp: 1234567890,
+    });
+    const opened_trade_title = 'Trade opened:';
+    const timestamp = 'Timestamp';
+
+    const renderTradeNotifications = (
+        mock_props: React.ComponentProps<typeof TradeNotifications> = {},
+        mock_store: ReturnType<typeof useStore> = mockStore({})
+    ) =>
+        render(
+            <StoreProvider store={mock_store}>
+                <TradeNotifications {...mock_props} />
+            </StoreProvider>
+        );
+
+    it('should not render notifications if show_trade_notifications is not passed', () => {
+        renderTradeNotifications();
+        expect(screen.queryByText(getCardLabels().TOTAL_PROFIT_LOSS)).not.toBeInTheDocument();
+        expect(screen.queryByText(getCardLabels().STAKE)).not.toBeInTheDocument();
+    });
+    it('should not render notifications if show_trade_notifications is not passed and has no notifications', () => {
```

Comment:
But you passed `show_trade_notifications` on the line below🤔

Reply:
i'll correct the title, it should be '_should not render notifications if show_trade_notifications is passed but has no notifications_'